yagpdb_docker/Dockerfile

```diff
@@ -1,47 +1,40 @@
 FROM golang:stretch as builder
 
-WORKDIR /appbuild/yagpdb/
-
-COPY go.mod .
-COPY go.sum .
-
+WORKDIR /appbuild/yagpdb
+COPY go.mod go.sum .
 RUN go mod download
-
 COPY . .
 
+WORKDIR /bin/yagpdb
+COPY cmd/yagpdb/posts posts/
+COPY cmd/yagpdb/static static/
+COPY cmd/yagpdb/templates templates/
+# Handle templates for plugins automatically
+COPY */assets/*.html templates/plugins/
+
 WORKDIR /appbuild/yagpdb/cmd/yagpdb
+RUN CGO_ENABLED=0 GOOS=linux go build -ldflags "-X github.com/jonas747/yagpdb/common.VERSION=$(git describe --tags)" -v -o /bin/yagpdb
+
 
-RUN CGO_ENABLED=0 GOOS=linux go build -ldflags "-X github.com/jonas747/yagpdb/common.VERSION=$(git describe --tags)" -v
 
 FROM alpine:latest
 
 WORKDIR /app
-VOLUME /app/soundboard \
+VOLUME \
+  /app/soundboard \
   /app/cert
 EXPOSE 80 443
 
 # We need the X.509 certificates for client TLS to work.
-# Also install tzdata, needed for Go timezone support, 
+# Also install tzdata, needed for Go timezone support,
 # and ffmpeg for soundboard support,
-# as the alpine image does not come with them by default. 
-RUN apk --no-cache add ca-certificates \
-  ffmpeg \
-  tzdata
-
-# Handle templates for plugins automatically
-COPY --from=builder /appbuild/yagpdb/*/assets/*.html templates/plugins/
-
-COPY --from=builder /appbuild/yagpdb/cmd/yagpdb/templates templates/
-COPY --from=builder /appbuild/yagpdb/cmd/yagpdb/posts posts/
-COPY --from=builder /appbuild/yagpdb/cmd/yagpdb/static static/
-
-COPY --from=builder /appbuild/yagpdb/cmd/yagpdb .
+# as the alpine image does not come with them by default.
+RUN apk --no-cache add \
+ca-certificates \
+ffmpeg \
+tzdata
```

Comment:
```suggestion
  ca-certificates \
  ffmpeg \
  tzdata
```
To be consistent with the rest of the multi-line instructions here.

Reply:
I would generally agree, but that is what I meant by "weird syntax on docker hub".
Docker hub lists the contents of the image layers in a single line. 
If we add the white space in front of the dependencies, then that would result in double whitespace in the detail pane on docker hub.
Changing it into a one-liner instead might be the better resolution with only 3 dependencies. 

Image of docker hub with white space:
![image](https://user-images.githubusercontent.com/69301522/122276776-6d35b100-cee5-11eb-9f5b-efec6ef8024c.png)

without white space:
![image](https://user-images.githubusercontent.com/69301522/122276856-82aadb00-cee5-11eb-835f-75ab4d63c1a3.png)

Reply:
Fair enough, TIL : )

Reply:
removing the white space at the end of the line instead would also work:
```suggestion
RUN apk --no-cache add\
 ca-certificates\
 ffmpeg\
 tzdata
```

or just write it all in one line:
```suggestion
RUN apk --no-cache add ca-certificates ffmpeg tzdata
```

Really not sure on that one

Reply:
Changed it into a one-liner